core-api/api_container_service.proto

```diff
@@ -165,3 +168,16 @@ message ExecCommandResponse {
   int32 exit_code = 1;
   bytes log_output = 2;
 }
+
+// ==============================================================================================
+//                                     Check Availability
+// ==============================================================================================
+message CheckAvailabilityArgs {
+  string service_id = 1;
+  string port = 2;
+  string path = 3;
```

Comment:
Can we add explanatory comments to each variable here, to make them more self-documenting?